wordpress-mobile/WordPress-iOS
# Capitalize title on Gutenberg

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/434

To test:

- Create a new post with Gutenberg
- Write a title, the first word should be capitalized

Update release notes:

- [x] If there are user facing changes, I have added an item to `RELEASE-NOTES.txt`. Internal changes only
